docs/rules/require-prop-comment.md

```diff
@@ -0,0 +1,180 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/require-prop-comment
+description: require that props have a comment
+---
+# vue/require-prop-comment
+
+> require that props have a comment
+
+- :exclamation: <badge text="This rule has not been released yet." vertical="middle" type="error"> ***This rule has not been released yet.*** </badge>
+
+## :book: Rule Details
+
+This rule enforces that every prop has a comment that documents it.
+
+<eslint-code-block :rules="{'vue/require-prop-comment': ['error']}">
+
+```vue
+<!-- ✓ GOOD -->
+<template>
+  <div>1</div>
+</template>
+<script>
+import { defineComponent } from '@vue/composition-api'
+
+export default defineComponent({
+  props: {
+    /**
+     * a comment
+     */
+    a: Number,
+    // b comment
+    b: Number,
+    // c
+    // comment
+    c: Number
+  }
+})
+</script>
+```
+
+</eslint-code-block>
+
+<eslint-code-block :rules="{'vue/require-prop-comment': ['error']}">
+
+```vue
+<!-- ✗ BAD -->
+<template>
+  <div>1</div>
+</template>
+<script setup>
+const props = defineProps({
+  a: Number,
+})
+</script>
+```
+
+</eslint-code-block>
+
+## :wrench: Options
+
+```json
+{
+  "vue/require-prop-comment": ["error", {
+    "type": "block"
+  }]
+}
+```
+- `type` ... Type of comment.Default is `"JSDoc"`
+  - `"JSDoc"` ... Only JSDoc comment are allowed,one is allowed.
+  - `"line"` ... Only line comment are allowed,one or more.
+  - `"block"` ... Only block comment are allowed,one is allowed.
+  - `"unlimited"` ... There is no limit to the number and type.
+
+### `"type":"block"`
+<eslint-code-block :rules="{'vue/require-prop-comment': ['error', {type: 'block'}]}">
+
+<!-- ✓ GOOD -->
+<script setup>
+const props = defineProps({
+  /*
+   * a comment
+   */
+  a: Number,
+})
+</script>
+
+<!-- ✗ BAD -->
+<script setup>
+const props = defineProps({
+  /*
+   * a comment
+   */
+  /*
+   * a other comment
+   */
+  a: Number,
+})
+</script>
```

Comment:
Why is this bad? There is a block comment before the `a` prop.

Reply:
block and JSDoc
1、Multiple lines are supported, and it is unnecessary to have multiple lines at the same time
2、The document generator may have difficulty recognizing which is what it needs

Reply:
I'd say this is an edge case that doesn't need to be handled in this rule. If the rule logic can be simplified by looking only if there is *a* comment for a prop (and not if there is *exactly one* comment), then we should rather do that.

Reply:
complete